inc/theme-options.php

```diff
@@ -0,0 +1,72 @@
+<?php 
+
+/**
+ * Set up a WP-Admin page for managing turning on and off theme features.
+ */
+function gutenbergtheme_add_options_page() {
+	add_menu_page(
+		'Theme Options',
+		'Theme Options',
+		'manage_options',
+		'gutenbergtheme-options',
+		'gutenbergtheme_options_page'
+	);
+
+	// Call register settings function
+	add_action( 'admin_init', 'gutenbergtheme_options' );
+}
+add_action( 'admin_menu', 'gutenbergtheme_add_options_page' );
+
+
+/**
+ * Register settings for the WP-Admin page.
+ */
+function gutenbergtheme_options() {
+	register_setting( 'gutenbergtheme-options', 'gutenbergtheme-dark-mode' );
+}
+
+
+/**
+ * Build the WP-Admin settings page.
+ */
+function gutenbergtheme_options_page() { ?>
+	<div class="wrap">
+	<h1><?php _e('Gutenberg Starter Theme Options', 'gutenbergtheme'); ?></h1>
+
+	<form method="post" action="options.php">
+		<?php settings_fields( 'gutenbergtheme-options' ); ?>
+		<?php do_settings_sections( 'gutenbergtheme-options' ); ?>
+
+			<table class="form-table">
+				<tr valign="top">
+					<td>
+						<label for="gutenbergtheme-dark-mode">
```

Comment:
The checkbox cannot be toggled by clicking the label. You should either remove the `for` attr here (and things will work because `<input>` is contained inside). Or you should add `id` attr to the input with the same value as its `name` and the `for` here.